rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/EvaluatorUtil.java

```diff
@@ -118,6 +112,28 @@ public static <T> InterruptibleFuture<T> runEvaluator(String task, CompletableFu
         });
     }
 
+    private static String formatMessage(Throw e) {
+        String message = "";
+        IValue thrown = e.getException();
+        if (thrown instanceof IString) {
+            message = ((IString)thrown).getValue();
+        }
+        else {
+            LimitedResultWriter res = new LimitedResultWriter(512);
+            try {
```

Comment:
This could be improved. The way the standard exceptions work from the `Exception` module in the
Stdlib is that must constructors of RuntimeException have a str field named "message"; it not then the name of the constructor is the message. Other fields contribute to the cause analysis. 

Proposal to translate this knowledge to a more elaborate error message over simply printing the constructor, since the end-user is the target audience rather than the rascal programmer. I would ignore the other causes for now to keep it short. The full error constructor can always be logged.

Reply:
good points, I've rewritten that code, would you review again @jurgenvinju ?